app/presenters/users/dashboard_presenter.rb

```diff
@@ -125,20 +119,20 @@ def course_details_array
       courses
         .includes(:communities)
         .map do |course|
-          {
-            id: course.id,
-            name: course.name,
-            review: course.id.in?(courses_with_review_access),
-            author: course.id.in?(courses_with_author_access),
-            enable_leaderboard: course.enable_leaderboard?,
-            description: course.description,
-            exited: student_dropped_out(course.id),
-            thumbnail_url: course.thumbnail_url,
-            linked_communities: course.communities.pluck(:id).map(&:to_s),
-            access_ended: student_access_end(course.id),
-            ended: course.ended?
-          }
-        end
+        {
+          id: course.id,
+          name: course.name,
+          review: course.id.in?(courses_with_review_access),
+          author: course.id.in?(courses_with_author_access),
+          enable_leaderboard: course.enable_leaderboard?,
+          description: course.description,
+          exited: student_dropped_out(course.id),
+          thumbnail_url: course.thumbnail_url,
+          linked_communities: course.communities.pluck(:id).map(&:to_s),
+          access_ended: student_access_end(course.id),
+          ended: course.ended?
+        }
+      end
```

Comment:
Incorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)